src/views/Nft/market/ActivityHistory/utils/sortActivity.tsx

```diff
@@ -50,11 +51,7 @@ export const sortActivity = ({
 
   const allActivity = [...transformAskOrders(askOrders), ...transformTransactions(transactions)]
   if (allActivity.length > 0) {
-    const sortedByMostRecent = allActivity.sort((activityItem1, activityItem2) => {
-      const timestamp1 = BigNumber.from(activityItem1.timestamp)
-      const timestamp2 = BigNumber.from(activityItem2.timestamp)
-      return timestamp2.sub(timestamp1).toNumber()
-    })
+    const sortedByMostRecent = orderBy(allActivity, (activity) => BigNumber.from(activity.timestamp), 'desc')
```

Comment:
I think orderBy can't handle BigNumber like that. There was a bug once in farms table where orderBy was passed BN in second argument and it looked like it worked but it was just doing random stuff. I believe it can only work with primitives (it could only work with custom classes if operator overloading was a thing in JS)
by the way why we even need BN for timestamp?

Reply:
Done, converted to number